cmd/backend/build_training_env/build-training-env.go

```diff
@@ -0,0 +1,78 @@
+package build_training_env
+
+/*
+$ your_cli_tool build-training-env --secret "YourTrainingSecretName" --bucket "YourTrainingBucketName"
+*/
+
+import (
+    "os"
+    "strings"
+    "fmt"
+    "log"
+    "github.com/aws/aws-sdk-go/aws"
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/secretsmanager"
+    "github.com/spf13/cobra"
+    "github.com/DSGT-DLP/Deep-Learning-Playground/cli/cmd/backend"
+)
+
+var secretNameTraining string // Name of the secret in AWS Secrets Manager
+var bucketNameTraining string // Name of the bucket for training
+
+var buildTrainingEnvCmd = &cobra.Command{
+    Use:   "build-training-env",
+    Short: "Build .env file for training/",
+    Run: func(cmd *cobra.Command, args []string) {
+        sess, err := session.NewSession(&aws.Config{Region: aws.String("us-west-2")})
+        if err != nil {
+            log.Fatal("Error creating AWS session: ", err)
+        }
+
+        smClient := secretsmanager.New(sess)
+        secretValue, err := smClient.GetSecretValue(&secretsmanager.GetSecretValueInput{SecretId: aws.String(secretNameTraining)})
+        if err != nil {
+            log.Fatal("Error retrieving secret: ", err)
+        }
+
+        path := "./training"
+
+        // Adding secrets to the .env file
+        writeToEnvFile(secretNameTraining, *secretValue.SecretString, path)
+
+        // Adding bucket name to the .env file
+        writeToEnvFile("BUCKET_NAME", bucketNameTraining, path)
+    },
+}
+
+func writeToEnvFile(paramName string, paramValue string, path string) error {
+    if err := os.MkdirAll(path, os.ModePerm); err != nil {
+        return fmt.Errorf("Error creating directory: %v", err)
+    }
+
+    content := strings.ToUpper(paramName) + "=" + paramValue
+    f, err := os.OpenFile(path+"/.env", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+    if err != nil {
+        return fmt.Errorf("Error creating .env file: %v", err)
+    }
+    defer f.Close()
+
+    fmt.Println("Writing to .env file", path)
+    n, err := f.WriteString(content + "\n")
+    if err != nil {
+        return fmt.Errorf("Error writing to .env file: %v", err)
+    }
+    fmt.Printf("Wrote %d bytes to .env file\n", n)
+
+    err = f.Sync()
+    if err != nil {
+        return fmt.Errorf("Error syncing .env file: %v", err)
+    }
+
+    return nil
+}
```

Comment:
Does this function of writing to `.env` file function the same across building `frontend-env`, `serverless-env`. If so, we can mitigate duplicate/redundant code by making a central `writeEnvFile` function

Reply:
i'll look onto this more and see if we can do this. off the top of my head though, By organizing this utility functions in a separate package like util, we can easily use them across multiple files in different folders. Just need to make sure to import the util package where needed.

Reply:
yeah. that was what my thought was, but you will need to take a look to see if that generalization can be made

Reply:
just tried this out, but facing issues on my end. Tried googling my way through it, but for some reason, system still doesn't recognize my envfile.go function. Weird. I'll circle back to this on Sunday. I've committed a utils folder though, and my `envfile.go` file for tracking. Best practice to have a `utils` folder at project root for these type of things @karkir0003

Reply:
Sounds good. That makes sense. Still thoroughly test that your change works

Reply:
got this to work! @karkir0003 

Dude it was literally because i didnt capatalize the first letter of the function. . In Go, the visibility of a name outside a package is determined by whether its first character is capitalized. To make the function visible (i.e., exported) so that it can be accessed from other packages, you need to start the function name with an uppercase letter.

this works now and is the most recent commit!